key.core.symbolic_execution/src/main/java/de/uka/ilkd/key/symbolic_execution/AbstractUpdateExtractor.java

```diff
@@ -560,7 +560,7 @@ protected Term createLocationPredicateAndTerm(
             argumentsList.add(param.createPreValueTerm());
             param.setValueTermIndexInStatePredicate(++argumentIndex);
         }
-        Term[] arguments = argumentsList.toArray(new Term[argumentsList.size()]);
+        Term[] arguments = argumentsList.toArray(new Term[0]);
```

Comment:
see comment on other commit: Why is this better?

Reply:
From the inspection description: 

>In older Java versions, using a pre-sized array was recommended, as the reflection call necessary to create an array of proper size was quite slow.
> However, since late updates of OpenJDK 6, this call was intrinsified, making the performance of the empty array version the same, and sometimes even better, compared to the pre-sized version. Also, passing a pre-sized array is dangerous for a concurrent or synchronized collection as a data race is possible between the size and toArray calls. This may result in extra nulls at the end of the array if the collection was concurrently shrunk during the operation.